README.md

```diff
@@ -100,10 +100,27 @@ Config options overview:
 | password | smtp    |                             | Password for email client.                                    |
 | host     | smtp    | smtp.gmail.com              | Host of the email client.                                     |
 
+### Transformation configuration
+
+Configuration files for TranSMART and cBioPortal must be placed in `transformation_config_dir`. 
+Specifically, the following are expected:
+
+- `sources_config.json` and `ontology_config.json`, described in [python_csr2transmart](https://github.com/thehyve/python_csr2transmart#usage); 
+the files reference the input data and need to be customized accordingly,
+- `portal.properties` file for cBioPortal; 
+the file  must match the mounted cBioPortal image version and server environment, 
+- `cbioportal_db_info` folder, containing configuration files for the cBioPortal database 
+(`cancertypes.json`, `genes.json`, `genesaliases.json`); 
+the files must match the mounted cBioPortal image version.
+ 
+Sample configuration files are provided in [test_data/test_data_NGS/config](https://github.com/thehyve/pmc-conversion/test_data/test_data_NGS/config).
+Be aware that `portal.properties` was created for a specific test environment
```

Comment:
This is true for all sample config files, so not sure if this information should be here. Also the portal.properties has too many options here. As we discussed, it can be limited to:
```
app.name=cbioportal

# database
db.user=cbio
db.password=<cBioPorta database password>
db.host=127.0.0.1:8306
db.portal_db_name=cbioportal
db.driver=com.mysql.jdbc.Driver
db.connection_string=jdbc:mysql://127.0.0.1:8306/
db.suppress_schema_version_mismatch_errors=false
db.use_ssl=false

# species and genomic information
species=human
ncbi.build=38
ucsc.build=hg38
~                      
```

Reply:
has the tilde (~) at the bottom a special meaning, or can I leave it out? I didn't notice it in the full version

Reply:
ah no, I added this accidentally, it should not be there